lib/errors.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+  let content = {};
```

Comment:
I'd use `const` here as content doesn't change. Generally I would always use `const` unless you need to mutate